controllers/clusteroperator_controller.go

```diff
@@ -54,14 +60,52 @@ type CloudOperatorReconciler struct {
 
 // Reconcile will process the cloud-controller-manager clusterOperator
 func (r *CloudOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	keys, err := r.PlatformOwner.List(ctx, r.Client)
+	if err != nil {
+		klog.Errorf("Unable to list platform objects %T: %+v", r.Object(), err)
+		return ctrl.Result{}, err
+	}
+
+	for _, platformPointer := range keys {
+		if err := r.sync(ctx, platformPointer); err != nil {
+			klog.Errorf("Unable to sync operands: %s", err)
+		}
+	}
 
 	if err := r.statusAvailable(ctx); err != nil {
 		klog.Errorf("Unable to sync cluster operator status: %s", err)
+		return ctrl.Result{}, err
 	}
 
 	return ctrl.Result{}, nil
 }
 
+func (r *CloudOperatorReconciler) sync(ctx context.Context, key client.ObjectKey) error {
+	resources, err := r.GetObjects(ctx, r.Client, key)
+	if err != nil {
+		return err
+	}
+
+	for _, resource := range resources {
+		if err = ApplyServerSide(ctx, r.Client, clusterOperatorName, resource); err != nil {
```

Comment:
May as well make this declare in a new scope as with the other if below
```suggestion
		if err := ApplyServerSide(ctx, r.Client, clusterOperatorName, resource); err != nil {
```